docs/DeveloperGuide.md

```diff
@@ -1247,3 +1247,44 @@ The time you entered does not exist. Please use a valid time in HH:MM format bet
 00:00 to 23:59.
 Example: addTransaction d=Bread n=John c=10 ts=25/06/2020 09:05 n=Self w=1
 ```
+
+### Better UI Representation for Long Fields
+
+- **Background**: Currently, we have `String` and `BigFraction` fields on our
+`Person`, `Transaction` and `Portion`. Some `String` fields do not necessarily
```

Comment:
Grammar: do not necessarily have a maximum limit

Reply:
Fixed